### PR TITLE
NM Caduceus (update)

### PR DESCRIPTION
**Pull request by ThatOneGoblin25**
## About the PR
- reworked pipes
- reworked wiring (SMES are looped to avoid AME flickering)
- reworked most rooms
- removed wallmounted nanomeds
- removed extra medkits (kept two regular ones)
- added 2 botany trays
- added crew monitoring computers & handheld
- added extra chemmaster and dispenser 
- added exosuit fabricator and robotech deluxe
- added medical medal
- added recreational room with TV, couch and arcade machine
- added virodrobe behind hidden door as a easter egg
- decals and other minor stuff

## Why / Balance
Ship that hasn't been touched in a while.

## Media
caduceus2

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- tweak: NM Caduceus has received an update.

### CODE REVIEW

**Comment by dvir001** [comment]
Ill be honest with you im not sure about microwave and hydro on this ship last time I tried to sell food for one they made fucking pizza.

**Comment by neuPanda** [comment]
minor changes i, as a frequent cad running medic, would like to see

remove cloning
add a jug or 2 of welding fuel, or a welding fuel tank

the aft extension feels like adding unnecessary entities

only 1 canister of AME fuel already loaded into the ame

2 morgues?

where is virology?
if the middle morgue is also suppose to second as surgery,  please make it a sterile room not one that has dirt everywhere

Thank you for collecting waste gasses and not just dumping them into space

note, i am not a maptainer, just providing my thoughts as somebody who frequently uses this ship

**Comment by ThatOneGoblin25** [comment]
> Ill be honest with you im not sure about microwave and hydro on this ship last time I tried to sell food for one they made fucking pizza.

Was unsure about the microwave since you can make advanced topicals, but I'll remove it. Botany is also something I'm "eh" on but it's needed for some of the higher tiers of chemistry, mainly necrosol.

**Comment by ThatOneGoblin25** [comment]
> remove cloning add a jug or 2 of welding fuel, or a welding fuel tank

Caduceus is the only civilian ship with a cloner, the other being the Empress, I think it's fine to keep until a medical poi is implemented. Welding fuel can easily be acquired from blue debris.

> the aft extension feels like adding unnecessary entities

It was to address some of the complaints about docking the ship. Now there's one side that is completely flat, should make things slightly easier. As I was working on this shape I felt like the shape is total wack but designing a new large medical ship is out of my scope for now.

> only 1 canister of AME fuel already loaded into the ame

There's already one loaded so I assume you mean the extra two outside? I will consider removing one.

> 2 morgues?

For morgue space, if responding to a mass casualty there can be 8+ dead bodies, although I suppose it's redundant since body bags stop rot timer, but those have to be purchased.

> where is virology? if the middle morgue is also suppose to second as surgery, please make it a sterile room not one that has dirt everywhere

No virology room, it's just a waste of a room. I kept virodrobe as a hidden easter egg tho.

**Comment by RadiantFlash7** [comment]
> @RadiantFlash7 hi letting you know there are some proposed changes to you ship.

Hey! Apologies for the lack of a response, I was out of town for over a week.

When I designed the caduceus, I explicitly left it with space for surgery and virology under the hope that upstream would eventually finish those, and we would have a ship able to make use of it right away when it was ported.
That said- Virology was a fair waste of space and I'm not too upset at seeing it gone.

Theres a few comments and such I'd like to make about this rework. The major positives I rather like: First being- with the pulse no longer around, the aft dock that was designed to act as a mothership dock area for ambulance ships isn't as important, as it was designed for the pulse in particular, as the searchlight didn't exist when the Cad was designed. So the dock change in the back is a fairly good idea.
The changes to engineering and atmos are fairly well done with an exception I'll hit on a bit later.
The addition of an exofab is interesting as well. I was hesitant to add one myself as to not step on a research ships toes, but having a medical cyborg is both useful and neat, so I rather like the addition of it.

I am however a bit more skeptical on some changes. 
Firstly, one of the chief concerns when designing the caduceus is that it would constantly produce enough food to indefinitely feed the crew, and also send food to others. I was very much worried of stepping on the toes of food producing ships that existed, and intentionally didn't provide a seed extractor. The original intent was to be able to provide the cad crew and their patients the ability to have some food on hand, but that it couldn't go on indefinitely, or atleast, not sell food the way a food ship could.

Secondly, the design change to move cryotubes further into the ship past medical storage confuses me somewhat. When set up, the cryotubes were an effective way to provide treatment, and probably should remain within the treatment center of the ship? In note of this- The changes to atmos, the gas mixer in particular, is not enough for the cad. Two oxygen canisters were provided in the original version due to the cryotubes needing a full canister to be able to operate. As the atmos is now- it is likely to eat up all of the spare air and leave none for fixing atmos onboard the cad.


As for the movement of cloning and the morgue- I'm ambivalent for the most part. From my understanding cloning may be on the chopping block- though moving the break room so far away from the botany section is not ideal. I am also not entirely sure on the purpose of the microwave in chemistry proper.

**Comment by ThatOneGoblin25** [comment]
Hi thank you for all this feedback, I've slept on this pr a bit because I wasn't quite happy with what I had done. Im not going to go over everything you addressed right now but I'll explain some changes I made.
First, botany changes as well as the microwave in chem was to facilitate the production of advanced topicals, you can cook chems, with poppies or aloe to get sutures and regenerative mesh. I was very skeptical of keeping botany at all but some plants are prerequisites for a few chems, particularly cryo ones. 
Second I wanted cryo to be separate from the treatment ward like on station medical departments, I think the centre is a good spot for it although I think now I should open up the space way more.
Third, cloning is on the chopping block but I decided to re-add it after taking it out, it was a bit of a afterthought to add it to the top but I think it has to stay until a hospital poi is implemented. I will rework morgue layouts and the cloning location can probably be swapped with surgery.

**Comment by dvir001** [comment]
I tried to fix conflicts on the file using webedit, need to actually see the map is working fine.

File: Resources/Maps/_NF/Shuttles/caduceus.yml [hunk 1]
```diff
@@ -1447,53 +1725,66 @@ entities:
...
+      pos: -7.5,9.5
+      parent: 1
+  - uid: 1149
     components:
     - type: Transform
       rot: -1.5707963267948966 rad
-      pos: 10.5,-13.5
+      pos: 3.5,8.5
       parent: 1
 - proto: AtmosDeviceFanTiny
```
Comment (PeccNeck) [inline Resources/Maps/_NF/Shuttles/caduceus.yml:1789]: Uh oh, tiny fans! Granted, this is somewhat of an older PR.

**Comment by dvir001** [comment]
Have to close this with the existing conflicts, this will have to be remade :/

**Pull request closed without merging**